### ISSUE TITLE
Cannot serialize result

### ISSUE THREAD

**Issue by grindfuk** [braintree/braintree_node#126]
### Description

Package Version: 2.6.0  
Node Version: Tried on both 4.6.1 and 8.5.0

If you stringify the result of the transaction: 

```
gateway.transaction.sale({
  amount: '5.00',
  paymentMethodNonce: 'nonce-from-the-client',
  options: {
    submitForSettlement: true
  }
}, function (err, result) {
  if (err) {
    console.error(err);
    return;
  }

  if (result.success) {
    console.log(JSON.stringify(result));
  } else {
    console.error(result.message);
  }
});
```

You will receive a TypeError

```
'TypeError: Converting circular structure to JSON',
     '    at Object.stringify (native)'
```

**Comment by crookedneighbor** [comment]
Ahh, this is a change we made in v2.6.0 that caused this. We'll make an update to workaround it, but in the until then, you should be able to use v2.5.0 without issue.

**Comment by grindfuk** [comment]
Awesome. Thanks. Downgrading works.

**Comment by jaypatel512** [comment]
Hey @grindfuk !

We just released 2.7.0 with the fix for this issue. Please update, and let us know if you run into any other issue. 

Thanks for pointing out the error.
